checks/users.js

```diff
@@ -0,0 +1,33 @@
+import '../server/env';
+
+import { sequelize } from '../server/models';
+// import models, { Op } from '../server/models';
+
+const check = true;
+// const fix = false;
+
+async function checkDeletedCollectives() {
+  const results = await sequelize.query(
+    `SELECT COUNT(*) as count
+     FROM "Users" u
+     LEFT JOIN "Collectives" c
+     ON c."id" = u."CollectiveId"
+     WHERE u."deletedAt" IS NULL
+     AND (c."deletedAt" IS NOT NULL OR c."id" IS NULL)`,
+    { type: sequelize.QueryTypes.SELECT, raw: true },
+  );
+
+  if (results[0].count > 0) {
+    if (check) {
+      throw new Error('No Users without a matching Collective');
```

Comment:
One closely related that I wanted to check today is cases of `Users` entries linked to non-`USER` Collectives. We have a few:

```sql
select c.slug, c.type
FROM "Users" u
INNER JOIN "Collectives" c ON u."CollectiveId" = c.id
WHERE c.type != 'USER'
AND c."deletedAt" IS NULL
AND u."deletedAt" IS NULL
```